### PR TITLE
DigitalOcean Docs / Rename DigitalOcean to dobs

### PR DESCRIPTION
**Pull request by akutz**
This patch adds documentation for the DigitalOcean Block Storage driver as well as renaming said driver to `dobs`.

### CODE REVIEW

File: .docs/user-guide/storage-providers.md [hunk 1]
```diff
@@ -656,14 +656,53 @@ libstorage:
...
+* Valid DigitalOcean account
+
+#### Configuration
+The following example illustrates a minimal configuration for the DigitalOcean
+block storage driver:
+
+```yaml
+dobs:
+  token: 123456
+  region: us-east-1
```
Comment (codenrhoden) [inline .docs/user-guide/storage-providers.md]: I'd prefer to have this be set to a real region that supports Volumes. Either `nyc1` or `sfo2`.

File: .docs/user-guide/storage-providers.md [hunk 2]
```diff
@@ -656,14 +656,53 @@ libstorage:
...
+
+!!! note
+    The DigitalOcean Block Storage driver currently only supports operating in
+    _local only_ mode where the libStorage server must be running on the same
+    host as the client.
+
+#### Requirements
+The DigitalOcean block storage driver has the following requirements:
+
+* Valid DigitalOcean account
```
Comment (codenrhoden) [inline .docs/user-guide/storage-providers.md:679]: I'd add that you need an access token, and here is the link/doc on how to generate one: https://www.digitalocean.com/community/tutorials/how-to-use-the-digitalocean-api-v2#how-to-generate-a-personal-access-token

**Review by codenrhoden — COMMENTED** [review]
Couple minor requests. Looks good!

File: .docs/user-guide/storage-providers.md (hunk 2 above)
Comment (akutz) [reply .docs/user-guide/storage-providers.md:679]: Handled! Thanks!

**Review by cduchesne — APPROVED** [review]

**Pull request merged**